resources/grafana/templates/dashboards/rhacs-central-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Let's add a comment to these template that explains why there is a `ConfigMap` and a `GrafanaDashboard` version.